editor/editor_file_system.cpp

```diff
@@ -532,7 +532,7 @@ bool EditorFileSystem::_update_scan_actions() {
 				String full_path = ia.dir->get_file_path(idx);
 				if (_test_for_reimport(full_path, false)) {
 					//must reimport
-					reimports.push_back(full_path);
+					reimports.append_array(_get_dependencies(full_path));
```

Comment:
Actually, shouldn't we still add `full_path` itself too, instead of only its dependencies?